python_examples/intervention_example/intervention_simulation_flow.py

```diff
@@ -26,8 +26,8 @@
 
 pop_params = {
     "population_size": 10000,
-    "cell_number": 200,
-    "microcell_number": 2,
+    "cell_number": 20,
```

Comment:
Make a separate example (place_closure_simulation_flow) for the place closure intervention. See the example code for case isolation. The example should show different number of cases when using different closure_household_infectiousness and/or closure_spatial_params.